examples/regtest/ph2/remd-gle/our_ref.pdb

```diff
@@ -0,0 +1,111 @@
+TITLE   cell{angstrom} Traj: positions{angstrom} Step:           0  Bead:       1
```

Comment:
Do we really want to keep PDB files in the examples?

Reply:
No strong feelings but I think most p-h2 examples use PDB. I'd say unrelated to this PR